maro/rl/policy/abs_policy.py

```diff
@@ -351,6 +360,7 @@ def _shape_check(
         self,
         states: torch.Tensor,
         actions: torch.Tensor = None,
+        **kwargs,
```

Comment:
Is `**kwargs` used in `_shape_check()`? Or you just want to keep it for future flexibility?

Reply:
Not used yet. Only for future flexibility.